From c46f6e80fca3d58d29341de73f75a0c50a491530 Mon Sep 17 00:00:00 2001
From: UtsavAgarwalADI <utsav.agarwal@analog.com>
Date: Thu, 29 May 2025 13:23:11 +0100
Subject: [PATCH] [PATCH 3/3] SC594-EZKIT: Adding softconfig changes for REV-D

Inverting the following to outputs:
       - EPROM
       - OSPI_CS (chip select)

Signed-off-by: UtsavAgarwalADI <utsav.agarwal@analog.com>
---
 arch/arm/boot/dts/adi/sc594-som-ezkit.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/adi/sc594-som-ezkit.dts b/arch/arm/boot/dts/adi/sc594-som-ezkit.dts
index e274ff11a1a7..95c3b7a2b1f3 100644
--- a/arch/arm/boot/dts/adi/sc594-som-ezkit.dts
+++ b/arch/arm/boot/dts/adi/sc594-som-ezkit.dts
@@ -33,14 +33,14 @@ ssw1: gpio@22 {
 
 		eeprom {
 			gpio-hog;
-			gpios = <0 0x0>;
+			gpios = <0 0x1>;
 			output-low;
 			line-name = "eeprom-en";
 		};
 
 		pushbutton {
 			gpio-hog;
-			gpios = <1 0x0>;
+			gpios = <1 0x1>;
 			output-low; /*output-high;*/
 			line-name = "pushbutton-en";
 		};
@@ -75,7 +75,7 @@ adau1979 {
 
 		octal {
 			gpio-hog;
-			gpios = <8 0x0>;
+			gpios = <8 0x1>;
 			output-low;
 			line-name = "octal-spi-cs-en";
 		};
-- 
2.34.1

